### PR TITLE
Line editing interface.mld

### PR DESCRIPTION
**Pull request by christinerose**
Mostly formatting, but there was also some syntax tightening (removing unnecessary words), so please let me know if anything is inaccurate.

### CODE REVIEW

File: doc/interface.mld [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
 We have a separate page describing the {{!page-ocamldoc_differences}markup differences from OCamldoc}.
 
 {2 CLI Interface}
 
-The way in which the [odoc] CLI is invoked is not trivial, and it requires careful
-ordering and correct arguments to produce correctly linked documentation. It’s not expected that
-end-users will invoke [odoc] by hand, but rather it will be driven by separate tools. As a consequence, it’s important to preserve the tools’ ability to create good documentation with
-each release of [odoc], so we’ll ensure backward compatibility of the CLI as much as possible.
+The way in which we invoke the [odoc] CLI is not trivial, and it requires careful
+ordering and accuarate arguments to produce correct linked documentation. It’s not expected that
```
Comment (panglesd) [inline doc/interface.mld:16]: Not sure if "accurate" is an adjective that applies to CLI arguments, but I think we understand the idea and it avoids the repetition! However, there is a typo:
```suggestion
ordering and accurate arguments to produce correct linked documentation. It’s not expected that
```

"correct link documentation" and "correctly linked documentation" feels different and the latter feels ~correcter~ more correct to me.

**Review by panglesd — APPROVED** [review]
One small comment!

File: doc/interface.mld (hunk 1 above)
Comment (christinerose) [reply doc/interface.mld]: Nice catch!!

**Comment by panglesd** [comment]
Thanks!

**Pull request merged**